gpkit/__init__.py

```diff
@@ -24,7 +24,23 @@
 
 # global variable initializations
 DimensionalityError = ValueError
-units = None
+ureg, units = None, None  # pylint: disable=invalid-name
+
+
+class GPkitUnits(object):
```

Comment:
should it inherit from `pint.unit.UnitRegistry`? This way we get any auxiliary methods/functionality that we aren't currently using but that pint super users might expect of a `ureg`